README.md

```diff
@@ -80,7 +80,7 @@ notification = Grocer::Notification.new(
   content_available: true                  # optional; any truthy value will set 'content-available' to 1
 )
 
-pusher.push(notification)
+pusher.push(notification) # return value is the number of bytes sent successfully
```

Comment:
Actually, I don't think that's a good idea.  I don't think the number of bytes sent is really useful information at all - the comment is just to avoid confusion as to what the return value is.  I don't mind either way, though.